variables.tf

```
@@ -16,6 +16,18 @@ variable "kms_key_id" {
   type        = string
 }
 
+variable "logging_target_bucket" {
+  default     = ""
```

I generally prefer to default to `null` as it's a more explicit "undefined" than an empty string.
```suggestion
  default     = null
```